app/Console/Commands/importSabbaticalEligibilty.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Library\UserService;
+use Illuminate\Console\Command;
+
+class importSabbaticalEligibilty extends Command
```

Comment:
Nit: I'm used to classes and their files being capitalized. I don't know if it impacts any Laravel functionality.
```suggestion
class ImportSabbaticalEligibilty extends Command
```